models/classes/class.QtiTestService.php

```diff
@@ -1096,7 +1083,7 @@ public function createContent(core_kernel_classes_Resource $test, $createTestFil
             $doc->documentElement->setAttribute('title', $test->getLabel());
 
             //generate a valid qti identifier
-            $identifier = tao_helpers_Display::textCleaner($test->getLabel(), '*', 32);
+            $identifier = Format::sanitizeIdentifier($test->getLabel());
```

Comment:
do we have a test for that?

Reply:
No, at least not yet.

Contributions are welcome ;)

Reply:
no no. it doesn't work like that :)) 
please add it if it is possible

Reply:
What do you mean, "it doesn't work like that"?

While it is important to have the test, especially since the behavior was buggy, it's more important to deliver the fix, and to work on the new features, requested by our customers. And if I to spend my time, working on a good test for this particular method, I may fail to deliver the rest.

Since we all work as a team, I invite you to contribute (because of the matters mentioned above). And it is your right to refuse and work on something else. But it certainly does work like that.

Reply:
Anyway. It seems like tomorrow I should be able to dedicate some time to that test.

Thanks for raising that concern!